### PR TITLE
Added more guide languages

### PR DESCRIPTION
**Pull request by samdark**

### CODE REVIEW

File: config/params.php [hunk 1]
```diff
@@ -86,7 +86,9 @@
             'pl' => 'Polski',       // Polish
             'pt-br' => 'Português brasileiro',  // Brazilian Portuguese
             'ru' => 'Русский',     // Russian
+            'uk' => 'Українська мова', // Ukrainian
```
Comment (cebe) [inline config/params.php]: isn't `мова` redundant here? or is there no single word like `Русский`?

File: config/params.php [hunk 2]
```diff
@@ -86,7 +86,9 @@
...
             'zh-cn' => '简体中文',  // Simplified Chinese
+            'vi' => 'Vietnamese', // Vietnamese
```
Comment (cebe) [inline config/params.php]: should this be `Tiếng Việt` ?

File: config/params.php (hunk 1 above)
Comment (samdark) [reply config/params.php]: @SilverFire?

**Pull request merged**